packages/eslint-config/.eslintrc.js

```diff
@@ -46,14 +45,45 @@ module.exports = {
     "import/no-cycle": ["error", { maxDepth: Infinity }],
     "import/no-useless-path-segments": "error",
     "import/no-relative-parent-imports": "error",
+    "import/no-deprecated": "error",
     "import/newline-after-import": "error",
+    "no-null/no-null": "error",
+    "react/no-danger": "error",
     "prettier/prettier": [
       "error",
       { trailingComma: "all" },
-      { usePrettierrc: false },
+      { usePrettierrc: false }
     ],
+    "@typescript-eslint/consistent-type-assertions": [
+      "error",
+      {
+        assertionStyle: "as"
+      }
+    ],
+    "@typescript-eslint/no-this-alias": "error",
+    "@typescript-eslint/no-extraneous-class": "error",
     "@typescript-eslint/no-use-before-define": ["error", { functions: false }],
     "@typescript-eslint/explicit-function-return-type": "off",
+    "@typescript-eslint/no-unused-vars": "error",
+    curly: ["error", "multi-line", "consistent"],
+    "prefer-const": "error",
+    "dot-notation": "error",
     "sort-imports": ["error", { ignoreDeclarationSort: true }],
-  },
+    "guard-for-in": "error",
+    "max-statements": ["error", 12],
```

Comment:
I believe allowing `ignoretoplevelfunctions` would give React functional components flexibility when applying this to the Jobber project. https://eslint.org/docs/rules/max-statements#ignoretoplevelfunctions